assembler/parts/header.html

```diff
@@ -0,0 +1,13 @@
+<!-- wp:group {"layout":{"type":"constrained"}} -->
```

Comment:
Adding the header as a pattern in a PHP file and then including it in the template part, as we do for the footer, would be better because that will allow us to add these comments:

```
<?php
/**
 * Title: header
 * Slug: assembler/header
 * Categories: featured, header
 * Block Types: core/template-part/header
 */
?>
```
The [header in TT4](https://github.com/WordPress/twentytwentyfour/blob/trunk/parts/header.html) doesn't add these comments. Do you have any idea why? @alaczek @richtabor

Reply:
> Adding the header as a pattern in a PHP file and then including it in the template part

But does that really matter for this theme, as this template part will be (almost) always overwritten by the header pattern that's selected in the Assembler flow?

Reply:
I'd say yes because:
- `Categories` is required to show that pattern in the right list in the inserter.
- `Block Types` is required to show that pattern in the modal when adding/replacing a header.

Reply:
Done!

Reply:
Template parts don't need the headers; only patterns do. :)